README.Rmd

```diff
@@ -790,9 +790,9 @@ Our goal in maintaining multiple lockfiles is to keep the list of dependencies r
 
 ### Using Lockfiles for Local Development
 
-When working on the model locally, you'll typically want to install non-core dependencies _on top of_ the core dependencies. To do this, simply run `renv::restore("<path_to_lockfile")` to install all dependencies from the lockfile.
+When working on the model locally, you'll typically want to install non-core dependencies _on top of_ the core dependencies. To do this, simply run `renv::restore(lockfile = "<path_to_lockfile")` to install all dependencies from the lockfile.
```

Comment:
This and the following docs change are typos I noticed while reminding myself how to install local dependencies. I tweaked them here to ensure the docs commands are correct.